vars_geoportailv3.yaml

```diff
@@ -103,5 +103,6 @@ interpreted:
         - closure_library_path
         cmd: ["node", "-e"]
 
-update_paths:
+update_paths:  # FIXME Why don't we have authtkt and checker in there?
```

Comment:
In the scaffold `update_paths` [includes](https://github.com/camptocamp/c2cgeoportal/blob/1.6/c2cgeoportal/scaffolds/create/vars_%2Bpackage%2B.yaml_tmpl#L113-L114) `authtkt` and `checker`. I don'k know what `update_paths` is so I did not know if I should add these as well.